src/components/HomePage/Banner.js

```diff
@@ -142,8 +143,11 @@ function renderInfo(classes) {
 					</Typography>
 				</Tooltip>
 			</Box>
-			{Date.now() < applyDeadline.getTime() &&
-				<Button disabled className={classes.apply} href={'https://forms.gle/7uokDycPQfU9B5oj8'} target='_blank'>
+			{
+				<Button
+					disabled={Date.now() < applicationOpen.getTime() || Date.now() > applyDeadline.getTime()}
```

Comment:
Instead of disabling this one, can you make it disappear instead?
We want the navbar one to be disabled, but not the banner one.